package.json

```diff
@@ -67,7 +67,8 @@
 		"trash": "^6.1.1",
 		"trash-cli": "^3.0.0",
 		"ts-node": "^8.5.3",
-		"typescript": "^3.8.2"
+		"typescript": "^3.8.2",
+		"yarn": "^1.22.4"
```

Comment:
You shouldnt have yarn here, this will be installed by the `yarn install`, but then you would need yarn already installed to install yarn :)
The requirement that yarn needs to be installed globally (of which there are a couple of ways)

Reply:
Could be useful for people who don't have yarn installed globally and attempt to do an `npm run`. Version should probably be as loose as possible though.

Reply:
> 
> 
> Could be useful for people who don't have yarn installed globally and attempt to do an `npm run`. Version should probably be as loose as possible though.

I removed the dependency and added a note in the README instead, seemed like a fair compromise :)